Templates/make_jenkins_EC2-Agent-Windows-instance.tmplt.json

```diff
@@ -0,0 +1,1613 @@
+            "Type": "String"
+        },
+        "AppVolumeDevice": {
+            "AllowedValues": [
+                "",
+                "xvdf",
+                "xvde",
+                "xvdg",
+                "xvdh",
+                "xvdi"
+            ],
+            "Default": "",
+            "Description": "(Optional) Device to mount an extra EBS volume. Leave blank to launch without an extra application volume",
+            "Type": "String"
+        },
+        "AppVolumeSize": {
+            "ConstraintDescription": "Must be between 1GB and 16384GB.",
+            "Default": "1",
+            "Description": "Size in GB of the EBS volume to create. Ignored if \"AppVolumeDevice\" is blank",
+            "MaxValue": "16384",
+            "MinValue": "1",
+            "Type": "Number"
+        },
+        "AppVolumeType": {
+            "AllowedValues": [
+                "gp2",
+                "io1",
+                "sc1",
+                "st1",
+                "standard"
+            ],
+            "Default": "gp2",
+            "Description": "Type of EBS volume to create. Ignored if \"AppVolumeDevice\" is blank",
+            "Type": "String"
+        },
+        "CfnEndpointUrl": {
+            "AllowedPattern": "^$|^http[s]?://.*$",
+            "Default": "https://cloudformation.us-east-1.amazonaws.com",
+            "Description": "(Optional) URL to the CloudFormation Endpoint. e.g. https://cloudformation.us-east-1.amazonaws.com",
+            "Type": "String"
+        },
+        "ChainScriptUrl": {
+          "AllowedPattern": "^$|^http[s]?://.*$|s3://.*$",
+          "Default": "",
+          "Description": "(Optional) URL to a \"chain\" script - a script to install add-ons not part of the core agent install.",
+          "Type": "String"
+        }, 
+        "ChainScriptParams": {
+          "Default": "",
+          "Description": "(Optional) Params to pass to chain load.",
+          "Type": "String"
+        }, 
+        "CloudWatchAgentUrl": {
+            "AllowedPattern": "^$|^s3://.*$",
+            "Default": "",
+            "Description": "(Optional) S3 URL to CloudWatch Agent installer. Example: s3://amazoncloudwatch-agent/windows/amd64/latest/AmazonCloudWatchAgent.zip",
+            "Type": "String"
+        },
+        "InstanceRole": {
+            "Default": "",
+            "Description": "(Optional) IAM instance role to apply to the instance(s)",
+            "Type": "String"
+        },
+        "InstanceType": {
+            "AllowedValues": [
+                "t2.micro",
+                "t2.small",
+                "t2.medium",
+                "t2.large",
+                "t2.xlarge",
+                "c4.large",
+                "c4.xlarge",
+                "m4.large",
+                "m4.xlarge",
+                "c5.large",
+                "c5.xlarge",
+                "m5.large",
+                "m5.xlarge"
+            ],
+            "Default": "t2.micro",
```

Comment:
Even if you choose to leave `micro` types in the list of alloweds, you _really_ want to default to something more useful than this value.